osquery/utils/azure/azure_util.h

```diff
@@ -0,0 +1,22 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#pragma once
+
+#include <osquery/utils/json/json.h>
+#include <osquery/utils/status/status.h>
+
+namespace osquery {
+
+std::string tree_get(JSON& doc, const std::string key);
```

Comment:
Do you mind naming this something more Azure-specific. `getAzureKey`?

Reply:
Yep, no problem!